modules/cluster-tagging.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="cluster-tagging_{context}"]
+= Tagging an existing cluster as owned
+
+A cluster uses the value of the `kubernetes.io/cluster/<clusterid>,Value=(owned|shared)` tag to determine which resources belong to the AWS cluster. These resources include:
+
+* All hosts.
+* All relevant load balancers to be used in the AWS instances.
+* All relevant security groups to be used with the AWS instances.
+* All EBS volumes.
```

Comment:
for EBS, no tags are required.

Reply:
For this item and the previous, this is a list of resources that are allocated based on the tagging of `owned` or `shared`. Would it be okay to leave them or just remove the whole list?

Reply:
@EricPonvelle I suggest to remove all of them, just add tag for instance host